compositions/upbound-aws-provider/elasticache/elasticache-redis.yaml

```diff
@@ -0,0 +1,230 @@
+          toFieldPath: spec.forProvider.region
+        - type: FromCompositeFieldPath
+          fromFieldPath: metadata.uid
+          toFieldPath: spec.forProvider.name
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.resourceConfig.providerConfigName
+          toFieldPath: spec.providerConfigRef.name
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.resourceConfig.tags
+          toFieldPath: spec.forProvider.tags
+          policy:
+            mergeOptions:
+              keepMapValues: true
+    - name: cluster-options
+      patches:
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.resourceConfig.nodeType
+          toFieldPath: spec.forProvider.nodeType
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.resourceConfig.replicasPerNodeGroup
+          toFieldPath: spec.forProvider.clusterMode[0].replicasPerNodeGroup
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.resourceConfig.numNodeGroups
+          toFieldPath: spec.forProvider.clusterMode[0].numNodeGroups
+        - type: FromCompositeFieldPath
+          fromFieldPath: spec.resourceConfig.multiAzEnabled
+          toFieldPath: spec.forProvider.multiAzEnabled
+  resources:
+    - name: redis-rg
+      connectionDetails:
+        - type: FromFieldPath
+          name: clusterArn
+          fromFieldPath: status.atProvider.arn
+        - type: FromFieldPath
+          name: clusterId
+          fromFieldPath: status.atProvider.id
+        - type: FromFieldPath
+          name: clusterAddress
+          fromFieldPath: status.atProvider.cacheNodes[0].address
+        - type: FromFieldPath
+          name: clusterPort
+          fromFieldPath: status.atProvider.cacheNodes[0].port
+      base:
+        apiVersion: elasticache.aws.upbound.io/v1beta1
+        kind: ReplicationGroup
+        metadata:
+          name: ec-redis-rg
+        spec:
+          forProvider:
+            description: "ec-redis-rg"
+            engine: redis
+            engineVersion: "7.0"
+            nodeType: cache.t4g.micro
+            clusterMode:
+              - numNodeGroups: 1
+                replicasPerNodeGroup: 0
+            multiAzEnabled: false
+            port: 6379
+            parameterGroupName: default.redis7.cluster.on
+            atRestEncryptionEnabled: true
+            transitEncryptionEnabled: true
+            automaticFailoverEnabled: true
+            subnetGroupNameRef:
+              name: subnetgroup-ec-cluster-via-composition
+            tags:
+              cluster: test-cluster
+              namespace: team-a
+              environment: dev
+              application: my-app
+              snapshotName:
+              from: composition
+            snapshotRetentionLimit: 0
+            snapshotWindow: "09:00-10:00"
+            applyImmediately: true
+            autoMinorVersionUpgrade: "true"
+            maintenanceWindow: "mon:10:40-mon:11:40"
```

Comment:
Can you move these defaults to the `definition.yaml` and then in the claim add a comment that there are defaults for there, and you can override them?